src/spatialNavigation.js

```diff
@@ -45,6 +45,12 @@ const THROTTLE_OPTIONS = {
   trailing: false
 };
 
+export const getNearestChild = (children) => {
```

Comment:
maybe just change this function name, it's not clear nearest to what :) 
I'd call it `getChildClosestToOrigin` if you agree

Reply:
Yeah, sure!)